examples/overrides/association/main.tf

```diff
@@ -7,33 +7,31 @@ provider "aws" {
 }
 
 locals {
-  identifier     = var.identifier # this is a random unique string that can be used to identify resources in the cloud provider
-  category       = "overrides"
-  example        = "association"
-  email          = "terraform-ci@suse.com"
-  name           = "tf-aws-server-${local.category}-${local.example}-${local.identifier}"
-  username       = "tf-ci-${local.identifier}"
-  image          = "ami-09b2a1e33ce552e68" # this must be an AMI in your region
-  public_ssh_key = var.key
-  key_name       = var.key_name
-  server_id      = var.server
+  identifier = var.identifier # this is a random unique string that can be used to identify resources in the cloud provider
+  category   = "overrides"
+  example    = "association"
+  email      = "terraform-ci@suse.com"
+  setup      = "tf-${local.category}-${local.example}-${local.identifier}"
+  name       = "tf-${local.category}-${local.example}-${local.identifier}-sut"
+  image      = "ami-09b2a1e33ce552e68" # this must be an AMI in your region
+  key_name   = var.key_name
+  server_id  = var.server
 }
 
-# selecting the vpc, subnet, and ssh key pair, generating a security group specific to the runner
-module "aws_access" {
+module "access" {
   source              = "rancher/access/aws"
   version             = "v1.1.1"
   owner               = local.email
-  vpc_name            = "default"
-  subnet_name         = "default"
+  vpc_name            = local.setup
+  subnet_name         = local.setup
   security_group_name = local.name
   security_group_type = "specific"
   ssh_key_name        = local.key_name
 }
 
```

Comment:
This example does not make use of provisioning, so skipping the ssh key generation both improves the test's speed and reliability. The source is not a path, it is an identifier in the terraform registry and is correct. The CIDR blocks have been selected intentionally and since they are internal and specific to the objects created in this test they have no implications outside of it.